.circleci/config.yml

```diff
@@ -60,4 +61,27 @@ workflows:
             branches:
               only:
                 - main
+      - orb-tools/publish-dev:
+          name: publish_dev_orb
+          attach-workspace: false
+          checkout: true
+          orb-name: ministryofjustice/cla-end-to-end-tests
+          orb-path: orb.yml
+          publish-token-variable: CIRCLE_TOKEN
```

Comment:
This is now the default value (the original circle ci context I was given to test with had the token under a different env var name, but I was able to specify this one when we got the new context). I think my preference would be to not include it, to make the extent of our overrides clearer.

Reply:
Updated .circleci/config.yml to reflect comment